aten/src/ATen/cuda/Atomic.cuh

```diff
@@ -348,6 +368,14 @@ GPU_ATOMIC_INTEGER(Mul, a * b, int16_t)
 GPU_ATOMIC_INTEGER(Mul, a * b, int32_t)
 GPU_ATOMIC_INTEGER(Mul, a * b, int64_t)
 
+inline __device__ c10::complex<float> gpuAtomicMul(c10::complex<float> *address, c10::complex<float> val){
```

Comment:
Looking at the failure and other function definitions, it looks like we need `__host__ __device__` here and for `complex_max` on 433, I can import and see whether this fixes the internal build tomorrow morning